sdk/purview/azure-purview-datamap/samples/get_types.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import os
+from azure.identity import ClientSecretCredential
+from azure.purview.datamap import DataMapClient
+
+logger = logging.getLogger("")
+logger.setLevel(logging.DEBUG)
+handler = logging.StreamHandler(stream=sys.stdout, )
```

Comment:
You need to import sys to fix this error: https://dev.azure.com/azure-sdk/public/_build/results?buildId=3528665&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=e3f8f308-6e5f-58aa-3a2a-fe425958de32&l=369 

similarly the CI errors are pointing out that LOG on line 23 is not defined